models/arrakis/arrakis_uniswap_pools.sql

```diff
@@ -0,0 +1,40 @@
+ {{
+  config(
+        schema='arrakis',
+        alias='uniswap_pools',
+        materialized = 'incremental',
```

Comment:
does this need to be incremental, if the underlying optimism one is? if so, can you apply if incremental logic? if not, remove to make a view